convert2rhel/systeminfo.py

```diff
@@ -53,6 +53,9 @@ def __init__(self):
         self.cfg_content = None
         self.system_release_file_content = None
         self.logger = None
+        # list of all enabled packages for Subscription Manager
+        # get using "subscription-manager repos --list-enabled" command
+        self.submgr_enabled_pkgs = []
```

Comment:
This is a list of repos, not packages
```suggestion
        # list of all enabled repositories for Subscription Manager
        self.submgr_enabled_repos = []
```